climada/hazard/base.py

```diff
@@ -1396,47 +1394,99 @@ def _read_att_excel(self, file_name, var_names):
                              f'{dfr.shape[0]} != {self.centroids.size}')
 
         self.intensity = sparse.csr_matrix(dfr.values[:, 1:num_events + 1].transpose())
-        self.fraction = sparse.csr_matrix(np.ones(self.intensity.shape,
-                                                  dtype=np.float))
+        self.fraction = sparse.csr_matrix(np.ones(self.intensity.shape, dtype=float))
 
-    def append(self, hazard):
-        """Append the events and centroids in hazard.
+    def append(self, *haz_list):
```

Comment:
This is a bit too much maybe, but `haz_list` is here not a list but a tuple.

Reply:
Solved in cd8720ef